concepts/60 Themes and Styles/05 Predefined Themes/50 Apply a Theme/05 Using SCSS.md

```diff
@@ -0,0 +1,47 @@
+You can import the theme bundle from the `bundles` folder from the `devextreme` package in the root `.scss` file:
+
+    /* @use 'devextreme/scss/bundles/dx.theme_name.scss';  */
+    @use 'devextreme/scss/bundles/dx.carmine.scss';
+
+Or you can import individual files for the whole theme. The example below imports files of the `Material Blue Light Compact` theme
+
+    @use 'devextreme/scss/widgets/material/colors' with ($color: 'blue', $mode: 'light');
+    @use 'devextreme/scss/widgets/material/sizes' with ($size: 'compact');
+    @use 'devextreme/scss/widgets/material/icons';
+    @use 'devextreme/scss/widgets/material/widget';
+    @use 'devextreme/scss/widgets/material/card';
+    @use 'devextreme/scss/widgets/material/fieldset';
+    @use 'devextreme/scss/widgets/material/common';
+    @use 'devextreme/scss/widgets/base/resizable';
+    @use 'devextreme/scss/widgets/base/draggable';
+    @use 'devextreme/scss/widgets/base/ui';
+    @use 'devextreme/scss/widgets/material';
+
+The theme path can be either `material` or `generic` depending on the theme family. The second solution is useful if you need to import only specific components. To do this, replace `@use 'devextreme/scss/widgets/material';` with `@use 'devextreme/scss/widgets/componentName';`:
+
+    @use "devextreme/scss/widgets/generic/colors" with ($color: "darkmoon");
+    @use "devextreme/scss/widgets/generic/sizes" with ($size: "default");
+    @use "devextreme/scss/widgets/generic/typography";
+    @use "devextreme/scss/widgets/generic/icons";
+    @use "devextreme/scss/widgets/generic/widget";
+    @use "devextreme/scss/widgets/generic/card";
+    @use "devextreme/scss/widgets/generic/fieldset";
+    @use "devextreme/scss/widgets/generic/common";
+    @use "devextreme/scss/widgets/base/resizable";
+    @use "devextreme/scss/widgets/base/draggable";
+    @use "devextreme/scss/widgets/base/ui";
+    @use "devextreme/scss/widgets/generic/textBox";
+    @use "devextreme/scss/widgets/generic/button";
+
+You can use the following variables in the imports:
```

Comment:
> You can use the following variables in **the imports**:

First, I'm going to question your article usage.

Second, I'm going to ask what "imports" is? What is that noun "import"? Import is a verb, not a noun. The only noun-like usage of "imports" is "imported goods". It's unlikely we can interpret imported files as such:

```suggestion
You can use the following variables in imported files:
```